subprojects/packagefiles/luajit/src/meson.build

```diff
@@ -0,0 +1,342 @@
+ljver = meson.project_version().split('.')
+ljver = '.'.join(ljver[0], ljver[1])
+abiver = '5.1'
+verdir = 'luajit-' + ljver
+jitdir = get_option('jitdir')
+if jitdir == ''
+  jitdir = get_option('datadir') / verdir / 'jit'
+endif
+
+foreach _var, _subdir : {
+  'install_cmod': 'libdir',
+  'install_lmod': 'datadir',
+}
+  _val = get_option(_var)
+  if _val != ''
+    assert(not fs.is_absolute(_val))
+  else
+    _val = get_option(_subdir) / 'lua' / abiver
+  endif
+  set_variable(_var, _val)
+endforeach
+
+luajit_h = custom_target(
+  'luajit.h',
+  capture: true,
+  command: [
+    'sed',
```

Comment:
Would be nice to avoid this external dependency. It may not always be available.

Reply:
There's also a dependency on `grep` through `unwind_check.sh`, so you need a Unix like system.

The alternative would be to try switching to Python for both cases. I don't think it's worth it.